dashboard/dashboard/templates/racetrack/portfolio.html

```diff
@@ -24,6 +24,7 @@ <h5 class="card-title">Fatmen Portfolio</h5>
                         <th>Family name</th>
                         <th>Version</th>
                         <th>Status</th>
+                        <th>Job type version</th>
```

Comment:
Fatman has its version, and there's version of its... job type. That's confusing for me (maybe I'm too far from RT now :) ), and for less technical users will probably also be. "What's a job within fatman?" they ask. Maybe you could add a on-hover description on how to simply understand that term.

Reply:
Agree, it might be confusing. WIll add the tooltip and I'll rename `Version` to `Fatman version`